### PR TITLE
Update NFTS Pallet Information

### PR DESCRIPTION
**Pull request by filippoweb3**
This is a second round of work on the new NFTs Pallet based on video tutorials provided by @sacha-l

### CODE REVIEW

File: docs/learn/learn-nft-pallets.md [hunk 1]
```diff
@@ -12,49 +12,55 @@ NFT-related actions within their dApp.
...
+is the admin. Then, under `config: PalletNftsCollectionConfig`, you can configure your collection's
+specifying different settings:
+
+- `settings` you can specify (in a bitflag-format) settings for your collection:
+  - Transferrable items: items can be non-transferrable (good for soul-bound NFTs),
+  - Unlocked metadata: metadata can be locked,
+  - Unlocked attributes: attributes can be locked,
+  - Unlocked max supply: allows to change the max supply until it gets locked (i.e. possibility to
+    change the supply for a limited amount of times),
+  - Deposit required.
```
Comment (jsidorenko) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
  - `Transferrable items`: when disabled, the items will be non-transferrable (good for soul-bound NFTs),
  - `Unlocked metadata`: when disabled, the metadata will be locked,
  - `Unlocked attributes`: when disabled, the attributes in the `CollectionOwner` namespace will be locked,
  - `Unlocked max supply`: allows to change the max supply until it gets locked (i.e. the possibility to
    change the supply for a limited amount of time),
  - `Deposit required`: when disabled, no mint deposit will be taken for items. This option can be set by a super-user only.
```

File: docs/learn/learn-nft-pallets.md [hunk 2]
```diff
@@ -12,49 +12,55 @@ NFT-related actions within their dApp.
...
+
+Everything is unlocked by default (bitflag `0`).
```
Comment (jsidorenko) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
Everything is unlocked by default (bitflag value `0`).
```

File: docs/learn/learn-nft-pallets.md [hunk 3]
```diff
@@ -12,49 +12,55 @@ NFT-related actions within their dApp.
...
+
+- `maxSupply: Option` (toggle option) gives you the possibility to specify the maximum number of
```
Comment (jsidorenko) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
- `maxSupply` (toggle option) gives you the possibility to specify the maximum number of
```

File: docs/learn/learn-nft-pallets.md [hunk 4]
```diff
@@ -64,8 +70,86 @@ to lock. Also, unauthorized and/or unprivileged transfers can be prevented by lo
...
+:::info
+
+Walkthrough video tutorial about minting an NFT.
 
+:::
+
+You can mint an NFT using the `nfts.mint` extrinsic. You must then specify the followings:
+
+- `collection`, the collection ID where you want to mint
+- `item`, the item ID (auto-incremented)
```
Comment (jsidorenko) [inline docs/learn/learn-nft-pallets.md]: technically, the item ID is not auto-incremented

**Comment by jsidorenko** [comment]
"Take ownership of an NFT from the [pallet-uniques](https://github.com/w3f/polkadot-wiki/blob/a5d31bbf2f017d3639432e2308e6b50157baa1b7/docs/learn/learn-nft-pallets.md#uniques-pallet)"

pls change pallet-uniques to pallet-nfts

**Comment by filippoweb3** [comment]
@DrW3RK removed video tutorial sections and pushed the latest commits (including grammar check). When we have the videos, these can be added to a separate PR. Let me know what you think!

File: docs/learn/learn-nft-pallets.md [hunk 5]
```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
...
 Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
 functionalities that are not needed in the runtime.
 
-Some features are common to the NFT world, such as:
+### Creating a Collection
+
+You can use the NFTs pallet to create NFTs collections. In the Polkadot-JS UI, go to Developer >
+Extrinsic and select the `nfts.create` extrinsic. When you create a collection, you must specify who
+the admin is. Then, under `config: PalletNftsCollectionConfig`, you can configure your collection's
+specifying different settings:
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
You can use the NFTs pallet to create NFT collections. In the Polkadot-JS UI, go to Developer >
Extrinsic and select the `nfts.create` extrinsic. When you create a collection, you must specify who
the admin is. Then, under `config: PalletNftsCollectionConfig`, you can configure your collection by 
specifying different settings:
```

File: docs/learn/learn-nft-pallets.md [hunk 6]
```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
...
+
+- `settings` you can specify (in a bitflag-format) settings for your collection:
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md:29]: I see that I can only enter a positive number `settings: u64` here. Let's say I would like to disable `Transferrable items` - what would be the numeric value for `settings: u64` ? As none of the settings below are visible on the UI, an example would make sense (on how to set the numeric value for settings).

File: docs/learn/learn-nft-pallets.md [hunk 7]
```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
...
+This can be useful for events such as Hackathons where participants who bought a ticket receive the
+NFT ticket from Collection A. Then, all holders of at least one item in Collection A (i.e. all
+ticket holders) will be given free avatar NFT from Collection X within the event schedule. After the
+event, any additional remaining items in Collection X can be made available to the public through a
+marketplace.
+
+The requirement to get the free avatar is being a holder of at least one NFT in Collection A. One
+can only claim the avatar specifying which NFT (i.e. the ID) owns in Collection A. The same NFT
+cannot be used twice, meaning that holders of multiple NFTs in Collection A (for example,
+participants to multiple Hackathons) can claim multiple avatars specific to each event.
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]: I made a few corrections. It is mentioned that same NFT cannot be used twice. But if someone has multiple NFTs, they should be able to claim multiple avatars right?

```suggestion
The requirement to get the free avatar is to hold at least one NFT in Collection A. One
can only claim the avatar specifying which NFT (i.e. the ID) they own in Collection A. The same NFT
cannot be used twice. Holders of multiple NFTs in Collection A (for example,
participants in multiple Hackathons) can claim multiple avatars specific to each event.
```

File: docs/learn/learn-nft-pallets.md [hunk 8]
```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
...
+
+:::
+
+After you minted an NFT, check which NFT IDs you own under which collection. In the Polkadot-JS UI
+go to Developer > Chain State > Storage, select the `nfts.account` extrinsic, and specify the
+account owning the NFT and the collection ID. You can also see all your collections by selecting the
+`collectionAccount` extrinsic.
 
+When a collection is destroyed, no one can pick up the collection ID again (including the owner).
+When a new collection is created, a new ID will be generated and assigned to that collection.
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]: Reordering chronological events

```suggestion
When a new collection is created, a new ID will be generated and assigned to that collection.
When a collection is destroyed, no one can pick up the collection ID again (including the owner).

```

File: docs/learn/learn-nft-pallets.md [hunk 9]
```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
...
+
+### Minting an NFT
+
+You can mint an NFT using the `nfts.mint` extrinsic. You must then specify the followings:
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
You can mint an NFT using the `nfts.mint` extrinsic. You must then specify the following:
```

**Review by DrW3RK — COMMENTED** [review]
Added some feedback. I went over the tutorial like an absolute beginner. I find it useful, but there are a few info gaps that need to be addressed.

File: docs/learn/learn-nft-pallets.md (hunk 6 above)
Comment (filippoweb3) [reply docs/learn/learn-nft-pallets.md:29]: Polkadot JS does not support bitflags. Settings are possible but at the moment only the default "0" is avaialble.

Comment (profradha) [reply docs/learn/learn-nft-pallets.md:29]: Got it! Let us add this as an info note here.

File: docs/learn/learn-nft-pallets.md [hunk 10]
```diff
@@ -37,8 +38,13 @@ specifying different settings:
   - `Deposit required`: when disabled, no mint deposit will be taken for items. This option can be
     set by a super-user only.
 
+:::info
+
+Note that currently Polkadot-JS does not support bitflags. Leave the settings field as it is.
 Everything is unlocked by default (bitflag value `0`).
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md:44]:
```suggestion
Note that currently, Polkadot-JS UI does not support bitflags. Leave the settings field as it is.
Everything is unlocked by default (bitflag value `0`).
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**